### PR TITLE
Add progress bar view

### PR DESCRIPTION
**Pull request by a2mark**

### CODE REVIEW

File: WeeGUI_MLI.s [hunk 1]
```diff
@@ -66,4 +66,6 @@ WGDisableMouse			= 64
 WGDeleteView			= 66
 WGEraseView				= 68
 WGExit					= 70
+WGCreateProgress		= 72
+
```
Comment (blondie7575) [inline WeeGUI_MLI.s:71]: Need to add the new WGSetState to the MLI

File: applesoft.s [hunk 2]
```diff
@@ -436,6 +436,65 @@ WGAmpersand_CHKBX:
...
+	
+	jsr WGPaintView
+	jsr WGBottomCursor
+
+	rts
+
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+; WGAmpersand_SETV
+; Set value (state)
```
Comment (blondie7575) [inline applesoft.s]: Let's make this more descriptive-  perhaps "Set the 'value' field of the selected view"

File: Documentation.md [hunk 3]
```diff
@@ -667,6 +690,19 @@ Not available
...
 
+####WGSetValue
+Sets the currently selected view's value. Currently only useful for progress bar views.
+
+<table width="100%">
+<tr><th>Assembly</th><th>Applesoft</th></tr><tr><td><pre>
+X:		WGSetState
+A: 		new value
+</td><td>
+&SETV(V value)
```
Comment (blondie7575) [inline Documentation.md]: The AppleSoft parameter docs don't need a pseudotype. "&SETV(value)" is sufficient.

File: weegui.s [hunk 4]
```diff
@@ -74,7 +74,7 @@ WGEntryPointTable:
 .addr WGDeleteView
 .addr WGEraseView
 .addr WGExit
-
+.addr WGCreateProgress
```
Comment (blondie7575) [inline weegui.s:77]: Need to add WGSetState

File: weegui.s [hunk 5]
```diff
@@ -86,10 +86,13 @@ WGInit:
 	;
 	; See section 5.1.4 in the ProDOS 8 Technical Reference Manual
 	; for an explanation of these values. We're reserving memory
-	; pages $7e-$95 so that ProDOS won't use our memory for file
+	; pages $7c-$95 so that ProDOS won't use our memory for file
```
Comment (blondie7575) [inline weegui.s]: $7b  . I'm going to reword this so this comment doesn't require so much maintenance.

**Comment by blondie7575** [comment]
Overall, looks great. Just a few small cleanup suggestions.

**Comment by a2mark** [comment]
Addressed all feedback. Please re-review.

**Pull request merged**